app/src/main/kotlin/cz/covid19cz/erouska/ui/mydata/MyDataVM.kt

```diff
@@ -31,12 +31,8 @@ class MyDataVM @ViewModelInject constructor(
 
     @OnLifecycleEvent(Lifecycle.Event.ON_RESUME)
     fun onResume() {
-        if (!DateUtils.isToday(prefs.getLastStatsUpdate())) {
```

Comment:
Je to ale trochu nebezpečné - pokud zjistíme v produkci, že máme zahlcené servery, nebudeme to umět vypnout. Co dát do RC s jakým delayem se mají data stahovat. Může se tam nastavit hodnota 0 = stahovat pokaždé. Ale půjde to změnit.

Reply:
Navrhuji spíše možnost RC konfigurovatelného fallbacku na předešlý stav : kdy dochází k update pouze tehdy pokud jsou k dnešnímu dni zastaralá data (atribut `date` není dnešní).
Jinak se stane, že pokud nastavíme delay např na 6 hodin tak ten kdo otevře appku minutu před tím než se aktualizuje BE, tomu se dalších 6 hodin nestáhnou nová data ani když už budou k dispozici. 

Zaroveň by se tímto RC flagem měl i vypnout SwipeToRefresh.

Reply:
Klidně, boolean feature flag je easy.

Reply:
FF `v2_updateNewsOnRequest`  zaintegrován.